ci_framework/roles/registry_deploy/tasks/main.yml

```diff
@@ -55,5 +55,5 @@
     dest: /etc/containers/registries.conf
     content: |-
       [[registry]]
-      location = "localhost:{{ cifmw_rp_registry_port }}"
+      location = "{{ cifmw_rp_registry_ip }}:{{ cifmw_rp_registry_port }}"
```

Comment:
im not sure if its related to this change but I'm now seign 
Error: trying to reuse blob sha256:e023e0e48e6e29e90e519f4dd356d058ff2bffbd16e28b802f3b8f93aa4ccb17 at destination: pinging container registry localhost:5001: Get "https://localhost:5001/v2/": http: server gave HTTP response to HTTPS client

https://review.rdoproject.org/zuul/build/07bb2baa7f4c4256ae4dfe98bb181fae/log/ci-framework/logs/ci_make_008_nova-operator_-_call_docker-push.log#18

Reply:
should this be "http://{{ cifmw_rp_registry_ip }}:{{ cifmw_rp_registry_port }}"